src/main/java/org/zalando/nakadi/service/subscription/zk/NewZkSubscriptionClient.java

```diff
@@ -111,9 +113,7 @@ private Topology readTopology() throws NakadiRuntimeException,
 
     private Topology parseTopology(final byte[] data) {
         try {
-            final Topology result = objectMapper.readValue(data, Topology.class);
-            getLog().info("Topology is {}", result);
```

Comment:
Topology is Topology

Reply:
@v-stepanov It's strange, but nothing will change :)